actions-server/package.json

```diff
@@ -4,7 +4,14 @@
   "description": "",
   "main": "src/index.js",
   "scripts": {
-    "start": "nodemon -r esm src/server.js"
+    "start": "yarn run serve",
+    "build": "yarn run build-ts",
+    "serve": "node build/actions-server/src/server.js",
+    "build-ts": "tsc",
+    "watch-ts": "tsc -w",
+    "debug": "npm run build && npm run watch-debug",
```

Comment:
I'm actually new to Yarn, but if we're going all-in on it, should this reference `yarn` instead of `npm`?

Reply:
doh. should all be yarn. nice catch